executor/runtime/docker/docker_linux.go

```diff
@@ -154,3 +159,35 @@ func cleanupCgroups(cgroupPath string) error {
 
 	return nil
 }
+
+func setupContainerNesting(parentCtx context.Context, c *runtimeTypes.Container, cred ucred) error {
+	if !c.TitusInfo.GetAllowNestedContainers() {
+		return nil
+	}
+	cgroupPath := filepath.Join("/proc/", strconv.FormatInt(int64(cred.pid), 10), "cgroup")
+	cgroups, err := ioutil.ReadFile(cgroupPath)
+	if err != nil {
+		return err
+	}
+	var ret error
+	for _, line := range strings.Split(string(cgroups), "\n") {
+		cgroupInfo := strings.Split(strings.TrimSpace(line), ":")
+		if len(cgroupInfo) != 3 {
+			continue
```

Comment:
log a `warn` here?

Reply:
this copy was largely copied from existing libcontainer cgroup parsing code. This a poor man's way of avoiding cgroup2. I don't see a reason to log that?